diagrams/library-imported-from-ecore.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
```

Comment:
Is this also generated? If yes, why do we need to commit it?

Reply:
Same thing as above.